downloads/github-git-advanced-cheat-sheet.md

```diff
@@ -0,0 +1,45 @@
+# Advanced Cheat Sheet
+
+
+## Merge commands
+
+Abort the merge
+
+```$ git merge --abort```
+
+## Stash commands
+
+Save a stash with a particular name
+
+```$ git stash save <stash name>```
+
+Delete a particular stash
+
+```$ git stash pop <stash name>```
```

Comment:
Yes, I meant the same :smile: 

Updated it with the fixes as you mentioned :+1: